### ISSUE TITLE
Many "404 File not found" errors for links in API docs

### ISSUE THREAD

**Issue by h-h-h-h** [moment/luxon#1175]
**Describe the bug**
In the API docs under https://moment.github.io/luxon/api-docs/index.html, there are many invalid links. This isn't exhaustive; it's just an example.

**To Reproduce**
- Visit https://moment.github.io/luxon/api-docs/index.html#datetime
- Click on the first link with text "DateTime#local". The link is: https://moment.github.io/luxon/api-docs/DateTime#local
- The following page says: "404 File not found..."

**Actual vs Expected behavior**
- The correct link would be: https://moment.github.io/luxon/api-docs/index.html#datetimelocal

**Desktop (please complete the following information):**

- OS: Windows
- Browser: Firefox
- Luxon version: N/A
- Your timezone: N/A

**Additional context**
&mdash;

**Comment by icambron** [comment]
Yeah, a lot of these broke when we changed doc systems, and I just haven't tracked them all down

**Comment by sam-s4s** [comment]
Was just about to post this issue. I'm in the process of switching over from moment to luxon, and I keep clicking on all the dead links :P

I'm happy to help update links if someone points me in the right direction?

**Comment by icambron** [comment]
@sam-s4s The doc files are here: https://github.com/moment/luxon/tree/master/docs

A good link looks like this:

```
[time zones documentation](zones.md?id=math-across-dsts)
```

which works both in Github and the rendered site

**Comment by sam-s4s** [comment]
Hmmm, this seems to have everything except the api docs?
https://moment.github.io/luxon/api-docs/index.html
I'm not sure where to find that.

**Comment by elliott-king** [comment]
It looks like all non-sidebar links to _static members_ are not working.

Example of **failing** links (static Datetime members within docs page):
<img width="695" alt="Screen Shot 2022-07-08 at 9 36 13 AM" src="https://user-images.githubusercontent.com/8884571/178003084-0b60e6eb-b569-4187-8fe4-530fd6e4df28.png">


Example of **failing** links (static Duration members within docs page):
<img width="624" alt="Screen Shot 2022-07-08 at 9 36 35 AM" src="https://user-images.githubusercontent.com/8884571/178003247-30c32fa6-8fe7-4e0a-b912-0f82425eee66.png">


Example of **successful** links (static Duration members in sidebar):
<img width="230" alt="Screen Shot 2022-07-08 at 9 37 12 AM" src="https://user-images.githubusercontent.com/8884571/178003281-ab1c0c4d-ca9d-4ddd-8ae2-7d3454bd36fc.png">


Example of **successful** links (instance Duration members within docs page): 
<img width="636" alt="Screen Shot 2022-07-08 at 9 37 52 AM" src="https://user-images.githubusercontent.com/8884571/178003300-7cc8db9c-0c12-408c-a632-00aecc38c671.png">

**Comment by pereriksson** [comment]
This is the last one I could find. I have gone through the API docs, I suggest we close this issue with this PR.

https://github.com/moment/luxon/pull/1767

**Comment by diesieben07** [comment]
@pereriksson Thank you for checking!
I did give it a glance as well and could not find any more issues except for the one you fixed, so I will close this issue for now.
